tests/test_users.py

```
@@ -509,6 +509,79 @@ def test_resend_invite(self):
+        )
+        self.user_id = create_response[0].user.id
+
+        try:
+            response = self.scalekit_client.users.assign_user_roles(
+                organization_id=self.org_id,
+                user_id=self.user_id,
+                role_ids=["member"]
+            )
+            self.assertEqual(response[1].code().name, "OK")
+            self.assertTrue(response[0] is not None)
+        except Exception as exp:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)